packages/ra-input-rich-text/src/index.tsx

```diff
@@ -45,7 +39,7 @@ interface Props {
     [key: string]: any;
 }
 
-const RichTextInput: FunctionComponent<Props> = props => {
```

Comment:
Would you mind renaming the props and exporting them too? Thanks!

Reply:
Done!